src/polymer/polymer2-element-scanner.ts

```diff
@@ -188,7 +188,7 @@ export class Polymer2ElementScanner implements JavaScriptScanner {
         description: (docs.description || '').trim(),
         superClass: this._getExtends(node, docs, warnings, document),
         mixins: jsdoc.getMixins(document, node, docs, warnings),
-        privacy: getOrInferPrivacy(className || '', docs, false),
+        privacy: getOrInferPrivacy(className || '', docs, false, 'private'),
```

Comment:
Scope is separate from privacy. We don't do scope correctly now, but there's a difference IMO between scope (what you can do) and privacy (what the author of the code wants you to do).